src/service/app/statisticsChart.ts

```diff
@@ -69,6 +74,17 @@ const handleStatistics = (wrapper: Response.Wrapper<State.StatisticsChart>) => {
   return datas
 }
 
+const handleStatisticsUnlceRate = (wrapper: Response.Wrapper<State.StatisticsChart>) => {
+  const { uncleRate: uncleRates } = wrapper.attributes
+  if (!uncleRates) return []
```

Comment:
A default value `[]` could be posted on `uncleRates`, like ` const { uncleRate: uncleRates = [] } = wrapper.attributes`

Reply:
OK

Reply:
Updated. https://github.com/nervosnetwork/ckb-explorer-frontend/pull/353/commits/3896cdba4af22bc1401f3678fb17ad27e153c367